### PR TITLE
add new colorful clock for haas example #1670

### PR DESCRIPTION
**Pull request by wushengtws**

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : 3c4a84e0314f3e448bed07c586e1fbb4a819d6e6

**Comment by skylarCai** [comment]
auto build success,number:141

**Comment by skylarCai** [comment]
autotest success,number:1351

**Review by ethanlcz — APPROVED** [review]

File: haas_lib_bundles/python/docs/examples/colorful_clock/esp32/README.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+
+ws2812Dev = None
+ws2812status = None
+
+#########
+# 物联网平台连接标志位
+iot_connected = False
+wlan = None
+
+# 三元组信息
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/colorful_clock/esp32/README.md:192]: no security risk

File: haas_lib_bundles/python/docs/examples/colorful_clock/esp32/code/main.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+ws2812Dev = None
+ws2812status = None
+
+#########
+# 物联网平台连接标志位
+iot_connected = False
+wlan = None
+
+# 三元组信息
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/colorful_clock/esp32/code/main.py:19]: no security risk

File: haas_lib_bundles/python/docs/examples/colorful_clock/esp32/README.md [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+productKey = "产品密钥" #需要填入物联网云平台申请到的productKey信息
+deviceName = "设备名称" #需要填入物联网云平台申请到的deviceName信息
+deviceSecret = "设备密钥" #需要填入物联网云平台申请到的deviceSecret信息
+
+# 物联网设备实例
+device = None
+
+# Wi-Fi SSID和Password设置
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/colorful_clock/esp32/README.md:200]: no security risk

File: haas_lib_bundles/python/docs/examples/colorful_clock/esp32/code/main.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+productKey = "产品密钥" #需要填入物联网云平台申请到的productKey信息
+deviceName = "设备名称" #需要填入物联网云平台申请到的deviceName信息
+deviceSecret = "设备密钥" #需要填入物联网云平台申请到的deviceSecret信息
+
+# 物联网设备实例
+device = None
+
+# Wi-Fi SSID和Password设置
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/colorful_clock/esp32/code/main.py:27]: pass

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**